docs/releasenotes/10.4.0.rst

```diff
@@ -30,6 +30,22 @@ The use in :py:mod:`.ImageCms` of input modes and output modes that are not Pill
 image modes has been deprecated. Defaulting to "L" or "1" if the mode cannot be mapped
 is also deprecated.
 
+16-Bit RGB/BGR Rawmodes
+^^^^^^^^^^^^^^^^^^^^^^^
+
+The following rawmodes have been deprecated and replaced with better-named rawmodes.
+
+RGB;15 ? XBGR;1555
```

Comment:
You originally had an arrow here. I don't think '?' is the best choice, and I'm guessing it wasn't intentional.

Reply:
Fixed. It looks like the file got converted to ASCII when I rebased it for some reason.

Reply:
Actually, would you mind copying the 'Deprecated' / 'Use instead' structure that we have already used elsewhere? https://pillow.readthedocs.io/en/stable/deprecations.html#imagecms-constants-and-versions-function

Reply:
done